src/test/java/ch/uzh/ifi/hase/soprafs21/controller/UsersApiControllerTest.java

```diff
@@ -86,4 +88,46 @@ void refreshToken() throws Exception {
         assertTrue(responseEntity.getHeaders().get("Set-Cookie").get(0).contains("mockRefreshToken"));
         assertEquals(mockAccessToken, responseEntity.getBody().getAccessToken());
     }
+
+    @Test
+    void logoutSuccess () throws Exception{
+        Long mockUserId = 1L;
+        given(userService.isRequesterAndAuthenticatedUserTheSame(mockUserId)).willReturn(Boolean.TRUE);
+
+        ResponseEntity<Void> responseEntity = usersApiController.usersLogoutPut(mockUserId);
+        assertEquals(HttpStatus.NO_CONTENT,responseEntity.getStatusCode());
```

Comment:
The actual call of UserService.userLogout is not checked in this test. 
To verify that userLogout was called by Service, something like this: verify(UserService.userLogout())

Reply:
added verify for userService.userLogout()